tweets/2023-10-11-0x3639-2.tweet

```diff
@@ -0,0 +1,3 @@
+---
+retweet: https://twitter.com/Learn_Zenon/status/1703736109698544011
```

Comment:
May want to remove the `retweet: ` part, otherwise that text will be in the tweet.

Reply:
Are you sure? I was following these directions:

https://github.com/zenon-network/twitter#retweeting-and-quote-retweeting

Reply:
I'll double check, but I had to remove those strings from the community thread because they were being rendered in-tweet.

Reply:
This is the official documentation: https://github.com/marketplace/actions/twitter-together

```
---
retweet: https://twitter.com/gr2m/status/1409601188362809349
---

twitter-together is awesome!
```

Reply:
maybe the `---` are required if you quote tweet.  I will try to remove them.